### PR TITLE
Fix deserialisation of PFSCapacityUpdate messages

### LINKED ISSUE #423: 'PFSCapacityUpdate' object has no attribute 'timestamp'
## Problem Definition

```
pfs-goerli_1            | 2019-07-03 11:42:45.245334 [critical ] Unhandled exception. Terminating the program...Please report this issue at https://github.com/raiden-network/raiden-services/issues [raiden_libs.gevent_error_handler]
pfs-goerli_1            | Traceback (most recent call last):
pfs-goerli_1            |   File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
pfs-goerli_1            |   File "/services/src/pathfinding_service/service.py", line 139, in _run
pfs-goerli_1            |     self._process_new_blocks(last_block)
pfs-goerli_1            |   File "/services/src/pathfinding_service/service.py", line 158, in _process_new_blocks
pfs-goerli_1            |     self.handle_event(event)
pfs-goerli_1            |   File "/services/src/pathfinding_service/service.py", line 184, in handle_event
pfs-goerli_1            |     self.handle_channel_opened(event)
pfs-goerli_1            |   File "/services/src/pathfinding_service/service.py", line 224, in handle_channel_opened
pfs-goerli_1            |     token_network_address=token_network.address, channel_id=event.channel_identifier
pfs-goerli_1            |   File "/services/src/pathfinding_service/database.py", line 269, in pop_waiting_messages
pfs-goerli_1            |     message.timestamp = message.timestamp.replace(tzinfo=timezone.utc)
pfs-goerli_1            | AttributeError: 'PFSCapacityUpdate' object has no attribute 'timestamp'
pfs-goerli_1            | Exiting...
pfs-goerli_1            | 2019-07-03 11:42:45.250418 [info     ] Stopping Pathfinding Service... [__main__]
```

## Actual Outcome

PFS crashes

## Expected Outcome

PFS shouldn't crash

### PR DESCRIPTION
**Pull request by palango**
Fixes #423

### CODE REVIEW

File: src/pathfinding_service/database.py [hunk 1]
```diff
@@ -265,8 +265,11 @@ def pop_waiting_messages(
             """,
             [to_checksum_address(token_network_address), hex256(channel_id)],
         ):
-            message: PFSFeeUpdate = JSONSerializer.deserialize(row["message"])
-            message.timestamp = message.timestamp.replace(tzinfo=timezone.utc)
+            message: DeferableMessage = JSONSerializer.deserialize(row["message"])
+
+            if isinstance(message, PFSFeeUpdate):
+                message.timestamp = message.timestamp.replace(tzinfo=timezone.utc)
```
Comment (Dominik1999) [inline src/pathfinding_service/database.py]: so what I understand here is that you replace the timestamp on the PFSFeeUpdate message with the current timestamp, right?

File: tests/pathfinding/test_database.py [hunk 2]
```diff
@@ -123,3 +137,64 @@ def test_feedback_stats(pathfinding_service_mock):
...
+        other_nonce=Nonce(1),
+        updating_capacity=TokenAmount(100),
+        other_capacity=TokenAmount(111),
+        reveal_timeout=50,
+        signature=EMPTY_SIGNATURE,
+    )
+    capacity_update.sign(LocalSigner(participant1_privkey))
+
+    for message in (fee_update, capacity_update):
+        database.insert_waiting_message(message)
```
Comment (Dominik1999) [inline tests/pathfinding/test_database.py]: here you insert 2 messages to the database

File: tests/pathfinding/test_database.py [hunk 3]
```diff
@@ -123,3 +137,64 @@ def test_feedback_stats(pathfinding_service_mock):
...
+
+        recovered_messages = list(
+            database.pop_waiting_messages(
+                token_network_address=token_network_address, channel_id=channel_id
+            )
+        )
+        assert len(recovered_messages) == 1
```
Comment (Dominik1999) [inline tests/pathfinding/test_database.py]: but only 1 message gets stored in the database, correct?

**Comment by palango** [comment]
Had to add some workarounds, those are tracked in https://github.com/raiden-network/raiden-services/issues/429

**Pull request merged**